frontend/src/components/messstelle/optionsmenue/panels/TagesTypRadiogroup.vue

```diff
@@ -66,7 +67,7 @@ function getTagesTypText(key: string): string | undefined {
 const helperText = computed(() => {
   let helpText = "";
   if (
-    chosenOptionsCopy.value.zeitraum.length === 2 &&
+    useDateUtils().isDateRange(chosenOptionsCopy.value.zeitraum) &&
```

Comment:
DateUtils außerhalb der Methode als Konstante anlegen und verwenden.

`const dateUtils = useDateUtils();`

Reply:
Angepasst. Siehe commit:

https://github.com/it-at-m/dave-frontend/pull/234/commits/9e5696b338cc97acdb25efa116bcdacb4ec1b610